webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.jsx',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'bundle.js',
+    publicPath: '/',
+  },
+  devtool: 'inline-source-map',
+  devServer: { contentBase: './dist' },
+  module: {
+    loaders: [
+      {
+        test: /\.jsx?$/,
```

Comment:
What does the `/\` means? :sweat_smile: 

didn't really found explanation about that in google :see_no_evil:

Reply:
yeah umm this is just regex, it's also how Webpack takes file extension tests. so let's say you want to also compile CSS files you would do a different test like this `test: /\.css$/,`  which again is testing for file names the **end** with `.css`